beacon-chain/utils/flags.go

```diff
@@ -55,4 +54,29 @@ var (
 		Name:  "genesis-json",
 		Usage: "Beacon node will bootstrap genesis state defined in genesis.json",
 	}
+	// EnablePOWChain tells the beacon node to use a real web3 endpoint.
+	EnablePOWChain = cli.BoolFlag{
+		Name:  "enable-powchain",
+		Usage: "Enable a real, web3 proof-of-work chain endpoint in the beacon node",
```

Comment:
Not sure if this `Enable` should be capitalized. It's all lower cases for flags below